src/olympia/amo/management/commands/compress_assets.py

```diff
@@ -65,8 +64,6 @@ def update_hashes(self):
             f.write('BUNDLE_HASHES = %s\n' % self.bundle_hashes)
 
     def handle(self, **options):
-        self.build_id = self.generate_build_id()
```

Comment:
😨 I don't think you meant to do just that - `self.build_id = '%s-%s' % (self.build_id, hex(int(time.time()))[2:])` needs to be changed too.

Reply:
> I don't think you meant to do just that - self.build_id = '%s-%s' % (self.build_id, hex(int(time.time()))[2:]) needs to be changed too.

Eh, I certainly didn't. Fix committed in abdc808